Detections/ASimNetworkSession/ExcessiveHTTPFailuresFromSource.yaml

```diff
@@ -6,7 +6,7 @@ description: |
 severity: Medium
 tags:
   - ParentAlert: https://github.com/Azure/Azure-Sentinel/blob/master/Detections/SophosXGFirewall/ExcessiveAmountofDeniedConnectionsfromASingleSource.yaml
-    version: 1.0.0
+    version: 1.0.1
```

Comment:
Please update the version number in line 63, not this one.

Reply:
Done

Reply:
Did you update the version in line 63 from 1.2.1 to 1.2.2? I can't see it. If you updated on your desktop, you need to push the commit. I often forget to do that :-)